sdk/keyvault/azure-security-keyvault-certificates/inc/azure/keyvault/certificates/certificate_client.hpp

```diff
@@ -115,6 +115,48 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
         CertificateCreateParameters const& parameters,
         Azure::Core::Context const& context = Azure::Core::Context()) const;
 
+    /**
+     * @brief Lists the certificate contacts for a specified key vault.
+     *
+     * @details The GetContacts operation returns the set of certificate contact
+     * resources in the specified key vault.
+     *
+     * @remark This operation requires the certificates/managecontacts permission.
+     *
+     * @param context The context for the operation can be used for request cancellation.
+     * @return The contacts list for the key vault certificate.
+     */
+    Azure::Response<std::vector<CertificateContact>> GetContacts(
```

Comment:
.Net returns an IList<CertificateContact>, i followed their model

Reply:
If the service returns pages as a response with continuation tokens, I would suggest looking at `PagedResponse<T>` rather than doing a direct language to language transcribe. I think some of the .NET KeyVault APIs were designed before azure core's page came in.

cc @heaths

Reply:
the service returns a list of contacts not paged